gobblin-cluster/src/main/java/org/apache/gobblin/cluster/GobblinHelixJobLauncher.java

```diff
@@ -221,7 +221,7 @@ protected void runWorkUnits(List<WorkUnit> workUnits) throws Exception {
     } finally {
       // The last iteration of output TaskState collecting will run when the collector service gets stopped
       this.taskStateCollectorService.stopAsync().awaitTerminated();
-      deletePersistedWorkUnitsForJob();
+      deleteWorkingDirectory();
```

Comment:
I think cleanupWorkingDirectory() would be a better name since we are only selectively deleting some items and not the whole directory.

Reply:
@htran1 Done